packages/tree-select/package.json

```diff
@@ -6,9 +6,9 @@
     "redux",
     "selector"
   ],
-  "author": "Ben Lowery <ben.lowery@automattic.com>",
+  "author": "Jake Fried <samouri@users.noreply.github.com>",
```

Comment:
Should code in Calypso repo really be "owned" by anyone like this or what's the context I might be missing? :-)